clients/admin-ui/src/features/integrations/configure-monitor/ConfigureMonitorModal.tsx

```diff
@@ -0,0 +1,42 @@
+import { Spinner, UseDisclosureReturn } from "fidesui";
+
+import AddModal from "~/features/configure-consent/AddModal";
+import ConfigureMonitorDatabasesForm from "~/features/integrations/configure-monitor/ConfigureMonitorDatabasesForm";
+import ConfigureMonitorForm from "~/features/integrations/configure-monitor/ConfigureMonitorForm";
+import { MonitorConfig } from "~/types/api";
+
+const ConfigureMonitorModal = ({
+  isOpen,
+  onClose,
+  formStep,
+  onAdvance,
+  monitor,
+}: Pick<UseDisclosureReturn, "isOpen" | "onClose"> & {
+  formStep: number;
+  monitor?: MonitorConfig;
+  onAdvance: (m: MonitorConfig) => void;
+}) => (
+  <AddModal
+    title={
+      monitor ? `Configure ${monitor.name}` : "Configure discovery monitor"
```

Comment:
Can we take this check one step further to ensure the name isn't an empty string?
```suggestion
      monitor?.name ? `Configure ${monitor.name}` : "Configure discovery monitor"
```